predicators/envs/grid_row.py

```diff
@@ -301,7 +307,10 @@ def _get_tasks(self, num: int,
                 },
                 self._door: {
                     "x": len(self._cells) // 2 + 0.5,
-                    "open": 0.0
+                    "open": 0.0,
+                    "target": 0.5,
+                    "open1": 0.0,
+                    "target1": 0.75
```

Comment:
could you name these with more informative names so it's easier to grok what they are + what's generally going on here? e.g. could you rename `open1` to `key_turn_angle` or something (that's what it corresponds to right?)